markdown/index.html

```diff
@@ -6,190 +6,121 @@
+    <script src="index.js"></script>
     <style>
       html, body, textarea {
-          width: 100%;
-          height: 100vh;
-          padding: 0;
-          margin: 0;
+        width: 100%;
+        height: 100vh;
+        padding: 0;
+        margin: 0;
+        font-family: sans-serif;
+        line-height: 1.4;
+      }
+      code {
+        background-color: #dfe9ff;
+        padding: 0 4px;
+        border-radius: 4px;
+        color: black;
+      }
+      blockquote {
+        border-left: 4px solid #ddd;
+        margin: 0;
+        padding: 0 1rem;
+        color: #666;
+      }
+      .editor-preview pre {
+        background-color: #eef3ff;
+        padding: 8px;
+        border-radius: 4px;
+      }
+      .editor-preview pre code {
+        background-color: unset;
+        padding: unset;
+        border-radius: unset;
+      }
+      hr {
+        margin-top: 2rem;
+        margin-bottom: 2rem;
+      }
+      .editor-preview table {
+        margin: 2rem 0;
+        border-spacing: 0;
+        border: 1px solid lightgrey;
+        border-left: none;
+        border-top: none;
+      }
+      .editor-preview table th, .editor-preview table td {
+        border: 1px solid lightgrey;
+        padding: 0.5rem 1rem;
+        border-right: none;
+        border-bottom: none;
+      }
+      .editor-preview table th:empty, .editor-preview table td:empty {
+        height: 0px;
+        padding: 0px;
+        border-top: none;
       }
       #error {
-          display: none;
-          background: red;
-          color: white;
-          padding: 20px;
-          text-align: center;
+        display: none;
+        background: red;
+        color: white;
+        padding: 20px;
+        text-align: center;
       }
       .editor-toolbar {
         border: none;
+        border-bottom: 1px solid #ddd;
+      }
+      .EasyMDEContainer {
+        height: 100%;
+        display: flex;
+        flex-direction: column;
       }
       .EasyMDEContainer .CodeMirror {
         border: none;
         border-radius: 0px;
-        border-top: 1px solid #ddd;
-        border-bottom: 1px solid #ddd;
+        flex: auto;
       }
-    </style>
-  </head>
-  <body>
-    <div id="error"></div>
-    <textarea id="txt"></textarea>
-    <script>
-      var tableId = null;
-      var rowId = null;
-      var colId = null;
-      var cachedData = null;
-      window.addEventListener('keypress', (ev) => {
-        // If user pressed Enter or Space
-        if (ev.keyCode === 13 || ev.keyCode === 32) {
-          // and we are in the read mode
-          if (txt.isPreviewActive()) {
-            // switch to edit mode.
-            editMode();
-            ev.preventDefault();
-            return;
-          }
+      .toolbar-read-mode {
+        position: absolute;
+        top: 4px;
+        right: 4px;
+        z-index: 100;
+        background-color: white;
```

Comment:
```suggestion
        background-color: #ffffff80;
```
What do you think about making it transparent a little bit, to show a text behind it when the first line is long enough?

Reply:
Good idea. I am thinking also of giving it a green border, so it's a bit more noticeable, wdyt?

<img width="485" alt="Screenshot 2023-01-19 at 12 23 45 PM" src="https://user-images.githubusercontent.com/1091143/213516050-b95bc838-fd6e-4b32-b050-2c4eb3f91594.png">

Reply:
Yes, it looks great.